tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/tests/TclCallParameterTest.xtend

```diff
@@ -0,0 +1,70 @@
+package org.testeditor.tcl.dsl.tests
+
+import javax.inject.Inject
+import org.eclipse.emf.ecore.EObject
+import org.eclipse.xtext.xbase.compiler.output.ITreeAppendable
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mock
+import org.testeditor.aml.dsl.tests.AmlModelGenerator
+import org.testeditor.dsl.common.testing.DummyFixture
+import org.testeditor.tcl.dsl.jvmmodel.AbstractTclGeneratorIntegrationTest
+import org.testeditor.tcl.dsl.jvmmodel.TclJvmModelInferrer
+
+import static org.mockito.Matchers.*
+import static org.mockito.Mockito.*
+
+class TclCallParameterTest extends AbstractTclGeneratorIntegrationTest {
+
+	@Inject TclJvmModelInferrer jvmModelInferrer // class under test
+	@Mock ITreeAppendable outputStub
+
+	@Inject extension AmlModelGenerator
+	@Inject extension TclModelGenerator
+
+	@Before
+	def void initMocks() {
+		when(outputStub.trace(any(EObject))).thenReturn(outputStub)
+		when(outputStub.append(any(CharSequence))).thenReturn(outputStub)
+	}
+
+	@Test
+	def void testCallParameterEscaping() {
+		// given
+		val amlModel = amlModel => [
```

Comment:
Can we extract this AML model to commons testing where the Dummy Fixture resides? Maybe just the textual AML description and parse it at this point.